src/Proto.Actor/ProcessRegistry.cs

```diff
@@ -47,12 +47,24 @@ public Process Get(PID pid)
             return aref;
         }
 
+        public (PID pid, bool ok) TryAdd(Process aref)
+        {
+            var id = NextId();
+            var pid = new PID
+            {
+                Id = id,
+                Address = Address
+            };
+            var ok = _localActorRefs.TryAdd(pid.Id, aref);
+            return (pid, ok);
+        }
+
         public (PID pid, bool ok) TryAdd(string id, Process aref)
         {
             var pid = new PID
             {
                 Id = id,

```

Comment:
Rather than have the above additional method can you just do: `Id = id ?? NextId();` ?